src/application/release-event-handler.ts

```diff
@@ -77,9 +77,6 @@ export class ReleaseEventHandler {
           const sourceTemplate = rulesetRepo.sourceTemplate(moduleRoot);
           const version = RulesetRepository.getVersionFromTag(tag);
           sourceTemplate.substitute({
-            OWNER: rulesetRepo.owner,
-            REPO: rulesetRepo.name,
-            VERSION: version,
```

Comment:
These substitutions are moved to where the value is most readily available. Allows more substitutions to be reused by the CLI code.